Remove the separator utility class and move its styling to .actionbar

yatriks opened the pull request.

parsakamali commented:
@yatriks so this is the way I previously had it. The reason I took the border off the actionbar and made it a separate class is so that we can make things span the full page width while having the content inside the row's container.  So this way the action bar can sit inside a row _inside_ a div with `.separator`, causing the lines to span the full page but the actionbar content to sit in the middle.

yatriks commented:
@parsakamali Gotcha gotcha.
In which case, I'd say the best solution is probably those border utility classes – that way you can do `<div class="row expanded border-gunmetal-500 border-bottom"><adk-actionbar /></div>`.

We'd have to add the `border-<direction>` classes, & those could be useful elsewhere too.

parsakamali commented:
@yatriks sure, I just want to be able to control the colors of the borders so that we're not hardcoding the color each time. If we can have 1-3 border color classes that would be used that way then I think it would give use finer control.

yatriks commented:
@parsakamali @expandrew:
I've modified the actionbar template to match Parsa's spec. See:
<img src="https://dl.dropbox.com/s/j5esxw83mze867l/Kapture%202017-04-11%20at%2017.07.06.gif" />

I also changed the references to the css files in `templates/`, so we can use `design-kit.arc.ht/templates`.

expandrew commented on scss/_adk-layout.scss:
```diff
@@ -9,8 +9,11 @@
 // Action Bar
 .actionbar {
   @include flex-align(null, middle);
+  @extend .mb-l;
```
@yatriks Gotta just include `margin-bottom: $space-l;` instead of `@extend`ing the other class.

This is what this does:
![image](https://cloud.githubusercontent.com/assets/3157928/24931220/832e94aa-1eda-11e7-82bd-0c2f5c3bef25.png)

yatriks commented:
@parsakamali: Fixed vertical alignment
@expandrew: Used explicit `margin-bottom` (good call)
